src/blockchain/ain/jellyfish/jellyfish.service.ts

```diff
@@ -102,20 +103,46 @@ export class JellyfishService {
   }
 
   async rawTxForSendFromLiq(to: string, amount: BigNumber): Promise<RawTxDto> {
-    // TODO (Krysh) implement raw tx
-    return {
-      hex: '',
-      scriptHex: '',
-      prevouts: [],
-    };
+    return this.rawTxForSend(Config.staking.liquidity.address, to, amount, true);
   }
 
   async rawTxForSendToLiq(from: string, amount: BigNumber): Promise<RawTxDto> {
-    // TODO (Krysh) implement raw tx
+    return this.rawTxForSend(from, Config.staking.liquidity.address, amount, false);
+  }
+
+  private async rawTxForSend(from: string, to: string, amount: BigNumber, addChange: boolean): Promise<RawTxDto> {
+    const network = this.getNetwork();
+
+    const [fromScript, fromPubKeyHash] = RawTxUtil.parseAddress(from, network);
+    const [toScript, toPubKeyHash] = RawTxUtil.parseAddress(to, network);
+
+    const unspent = addChange
+      ? await this.whaleClient.getAllUnspent(from)
+      : await this.whaleClient.getUnspent(from, amount);
+    const [prevouts, total, scriptHex] = RawTxUtil.parseUnspentUntilAmount(unspent, amount);
+
+    const vins = RawTxUtil.createVins(prevouts);
+    const vouts = [RawTxUtil.createVoutReturn(toScript, amount)];
+    if (addChange) {
+      const change = RawTxUtil.createVoutReturn(fromScript, total.minus(amount));
+      vouts.push(change);
+    }
+    const witnesses = [
+      RawTxUtil.createWitness([
+        RawTxUtil.createWitnessScript(fromScript, fromPubKeyHash),
+        RawTxUtil.createWitnessScript(toScript, toPubKeyHash),
+      ]),
+    ];
+
+    const tx = RawTxUtil.createTxSegWit(vins, vouts, witnesses);
+    const fee = calculateFeeP2WPKH(new BigNumber(0.00001), tx);
```

Comment:
Is 0.00001 the fee rate?

Reply:
yes the lowest possible. should we move it into the config? I think it will never change

Reply:
No, is ok 👍